frontend/components/GroupCardComponent/index.js

```diff
@@ -1,15 +1,20 @@
 import React, { useState} from 'react';
 import { View } from 'react-native';
-import { Avatar, Card, IconButton } from 'react-native-paper';
+import { Avatar, IconButton, List, Divider } from 'react-native-paper';
 import styles from './styles';
 import {VIEW_GROUP} from '../../navigation/tab_navigator/stacks/groups/screen-names';
 
-
-
 function CardComponent({navigation, groupName, groupId, userId, heartActiveCallback, index, heartStatus=false, groupDescription="Buenas Tardes Amigo"}) {
```

Comment:
You should also change the name of this function and file to be "ListItemComponent" and "GroupListItemComponent"